### PR TITLE
Submit question and Save it

### PR DESCRIPTION
**Pull request by tongweiz**
Implement submit question and save it, then redirect to homepage and show the question.
Notice: Only logged people can do ask a question!

### CODE REVIEW

File: soen341-UB3-QA/resources/views/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```
Comment (irinafrazao) [inline soen341-UB3-QA/resources/views/index.blade.php]: This view is not used anywhere. It is not needed either. You can remove this file.

File: soen341-UB3-QA/resources/views/ask.blade.php [hunk 2]
```diff
@@ -36,21 +36,26 @@
...
                 <div class="card-body">
-                    <form>
+                    {{--<form>--}}{!! Form::open(['url' => 'ask']) !!}
                         <div class="form-group">
-                            <h4 class="mt-4">Title:</h4>  
-                            <textarea class="form-control" rows="1"></textarea>
-                        </div> 
-                    </form>
+                            {!! Form::label('title','Title:') !!}
+                            {!! Form::text('title', null, ['class' => 'form-control', 'placeholder' => 'Titles...']) !!}
```
Comment (irinafrazao) [inline soen341-UB3-QA/resources/views/ask.blade.php]: "Titles" is a bad placeholder value. Try something like "Enter your question title here"

File: soen341-UB3-QA/resources/views/ask.blade.php [hunk 3]
```diff
@@ -36,21 +36,26 @@
...
                 <div class="card-body">
-                    <form>
+                    {{--<form>--}}
                         <div class="form-group">
-                            <h4 class="mt-4">Contents:</h4>  
-                            <textarea class="form-control" rows="10"></textarea>
+                            {{--<h4 class="mt-4">Contents:</h4>--}}
+                            {{--<textarea class="form-control" rows="10"></textarea>--}}
+                            {!! Form::label('content', 'Contents:') !!}
+                            {!! Form::textarea('content', null, ['class' => 'form-control', 'placeholder' => 'Contents...']) !!}
```
Comment (irinafrazao) [inline soen341-UB3-QA/resources/views/ask.blade.php]: "Contents" is a bad placeholder value. Try something like "Enter the details of your question here"

**Review by irinafrazao — CHANGES_REQUESTED** [review]
Small changes. Index view not needed. English mistakes in placeholder values

**Comment by Alex44711** [comment]
Implement submit question and save it, then redirect to homepage and show the question.

**Pull request merged**